command/ibft_propose.go

```diff
@@ -59,32 +64,27 @@ func (p *IbftPropose) Synopsis() string {
 
 // Run implements the cli.IbftPropose interface
 func (p *IbftPropose) Run(args []string) int {
-	flags := p.FlagSet("ibft propose")
+	flags := p.FlagSet(p.GetBaseCommand())
 
-	var add, del bool
-	flags.BoolVar(&add, "add", false, "add")
-	flags.BoolVar(&del, "del", false, "del")
+	var vote bool
+	var ethAddress string
+
+	flags.BoolVar(&vote, "vote", false, "")
```

Comment:
If we want to keep it within a bool, it needs to be either `true` or `false`, and in Golang it's false by default.
I'll change it to `true`, since it feels more natural.
@lazartravica thoughts?

Reply:
Updated the vote so it takes in a string (`add` or `remove`) 👍

Reply:
@zivkovicmilos Looks good, only comment is that in the eip the vote types are named "auth" and "drop" https://github.com/ethereum/EIPs/issues/650, not add or remove

Reply:
As for the argument being mandatory I agree completely 👍

Reply:
This is even better. We should try to stick to the EIP naming conventions as much as possible